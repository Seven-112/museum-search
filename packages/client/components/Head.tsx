import "bootstrap/dist/css/bootstrap.min.css";
import "leaflet/dist/leaflet.css";
import NextHead from "next/head";

interface HeadProps {
  title: string;
}

export function Head({ title }: HeadProps) {
  return (
    <NextHead>
      <meta charSet="UTF-8" />
      <title>{title}</title>
      <meta name="viewport" content="width=device-width, initial-scale=1" />
      <link rel="icon" href="/static/favicon.ico" />
    </NextHead>
  );
}
