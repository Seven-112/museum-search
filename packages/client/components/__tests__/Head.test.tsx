import { shallow } from "enzyme";
import { Head } from "../Head";
import toJson from "enzyme-to-json";

describe("Head component", () => {
  it("Renders.", () => {
    const wrapper = shallow(<Head title="Page Title" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
