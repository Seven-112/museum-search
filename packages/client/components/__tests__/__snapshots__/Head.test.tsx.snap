// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head component Renders. 1`] = `
<SideEffect(Head)>
  <meta
    charSet="UTF-8"
  />
  <title>
    Page Title
  </title>
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />
  <link
    href="/static/favicon.ico"
    rel="icon"
  />
</SideEffect(Head)>
`;
